### PR TITLE
implementations to use TEASERs new model report

### PR DESCRIPTION
**Pull request by DaJansenGit**
This integrates TEASERs new model report functionality by default for bim2sim

### CODE REVIEW

File: bim2sim/elements/bps_elements.py [hunk 1]
```diff
@@ -1751,6 +1751,23 @@ def _get_building_name(self, name):
...
+        """Calculates the average height of all storeys."""
+        storey_height_sum = 0
+        for storey in self.storeys:
+            if storey.height:
+                height = storey.height
+            elif storey.gross_height:
+                height = storey.gross_height
+            elif storey.net_height:
+                height = storey.net_height
+            storey_height_sum += height
```
Comment (veronikarichter) [inline bim2sim/elements/bps_elements.py:1767]: please add an option for cases, where none of these conditions are met (add an else statement).

File: bim2sim/elements/bps_elements.py [hunk 2]
```diff
@@ -1751,6 +1751,23 @@ def _get_building_name(self, name):
...
+        avg_height = storey_height_sum / len(self.storeys)
```
Comment (veronikarichter) [inline bim2sim/elements/bps_elements.py:1768]: Are storeys mandatory? otherwise len(self.storeys) could be zero.

**Review by veronikarichter — CHANGES_REQUESTED** [review]

File: bim2sim/elements/bps_elements.py (hunk 1 above)
Comment (DaJansenGit) [reply bim2sim/elements/bps_elements.py]: improved this with last commit

**Comment by DaJansenGit** [comment]
I made the asked changes and added an additional commit which created a problem in TEASER model report. Can we merge this?

**Review by veronikarichter — APPROVED** [review]

**Pull request merged**